spec/fixtures/fixtures.rb

```diff
@@ -86,7 +86,7 @@ def person(data)
 end
 
 Contract Proc => Any
-def call(&blk)
+def do_call(&blk)
```

Comment:
Not the best idea to define `call` on `Object`. It mess with `rspec-mocks`, which makes it run into stack level too deep.